messenger.go

```diff
@@ -292,6 +304,16 @@ func (m *TGMessenger) ProcessMessage(message *tgbotapi.Message, chat *Chat) {
 	}
 }
 
+func (m *VKMessenger) getSenderName(message object.MessagesMessage) string {
+	userResponse, err := m.vk.UsersGet(api.Params{"user_ids": message.FromID})
+	if err != nil {
+		log.Print("could not find vk user with id ", message.FromID, err)
+		return ""
+	}
+	// have not found a way to get chat name
+	return concatenateMessageSender(userResponse[0].FirstName+" "+userResponse[0].LastName, "vk")
```

Comment:
we still cannot get vk chat name, can we?

Reply:
VK gives an error when trying to call getChat with the chat's id. It would be great if you can solve it, but for now I don't have any ideas